server/models/studentModels.js

```diff
@@ -1 +1,33 @@
-const mongoose = require('mongoose');
\ No newline at end of file
+const mongoose = require('mongoose');
+
+// create a constructor 
+const Schema = mongoose.Schema;
+
+// create a student schema
+const studentSchema = new Schema({
+  firstName: {type: String, required: true},
+  lastName: {type: String, required: true},
+  userName: {type: String, required: true, unique: true},
+  password: {type: String, required: true},
+});
+
+// create a collection instance of the model
+const Student = mongoose.model('students', studentSchema);
+
+
+// create a goal schema
+const goalsSchema = new Schema({
+  description: {type: String, required: true}, 
+  dueDate: Number,
```

Comment:
change dueDate value to be (instance of) Date

Reply:
done